recipe/meta.yaml

```
@@ -1,61 +1,52 @@
-{% set version = "4.4.1" %}
+{% set version = "7.0.0" %}
 
 package:
   name: docker-py
   version: {{ version }}
 
 source:
   url: https://github.com/docker/docker-py/archive/{{ version }}.tar.gz
-  sha256: e28b9033c165e622562782691234bd1ac2da13d59bc60b389ecb9b7a3d297d43
+  sha256: 6a82c1630219a035c1717cf11082e97d109689c5e7f2da4aaabea4bb85a74faa
 
 build:
-  number: 5
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
   host:
     - python
     - pip
+    - setuptools
```

```suggestion
    - setuptools
    - setuptools_scm
```
https://github.com/docker/docker-py/blob/7d8a161b12f4c5e16776c57bfc5ce7077f404995/pyproject.toml#L2C31-L2C45

`setuptools_scm` is required in `host`. Because without it I see in logs now:
`Successfully installed docker-0.0.0`

---

Thank you!

---

Hm... an error:
```
LookupError: setuptools-scm was unable to detect version for /croot/docker-py_1715788347113/work.

Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.

```
I guess the versioning of docker-py is broken